models/db.go

```diff
@@ -0,0 +1,49 @@
+package models
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+)
+
+const (
```

Comment:
Instead of being defined as constants, these values should be read from environment variables so that we can configure them to be whatever we want at run-time.